src/module.c

```diff
@@ -6043,6 +6046,11 @@ RedisModuleClient *RM_CreateModuleClient(RedisModuleCtx *ctx) {
     return rmc;
 }
 
+/* Releases a RedisModuleClient.
+ *
+ * Note that the client must not be released if it has an outstanding command running.
+ * A blocked RM_Call (using the 'K' flag), requires calling RM_CallReplyPromiseAbort
+ * first, and wait for the unblock handler to be called. */
```

Comment:
The unblock handler will not be called if the abort succeeded.  maybe change it to:
```
requires calling RM_CallReplyPromiseAbort successfully or wait for the unblock handler to be called.
```